pkg/watch/health.go

```diff
@@ -36,6 +36,7 @@ const TTL = 60 * time.Second
 // has a running MonitorHealth go routine
 type PodWatch struct {
 	manifest pods.Manifest
+	client   *http.Client
```

Comment:
hmm, it doesn't seem to me like having this declaration is terribly necessary. Do you have something in mind I'm not thinking of?